packages/lesswrong/lib/collections/users/newSchema.ts

```diff
@@ -1672,7 +1672,13 @@ const schema = {
   notificationRepliesToMyComments: {
     database: {
       type: "JSONB",
-      defaultValue: defaultNotificationTypeSettings,
+      defaultValue: {
```

Comment:
I think this should just be replaced by `bothChannelsEnabledNotificationTypeSettings`